awx_collection/test/awx/test_credential.py

```diff
@@ -78,13 +78,13 @@ def test_create_custom_credential_type(run_module, admin_user):
 
     ct = CredentialType.objects.get(name='Nexus')
     result.pop('invocation')
+    result.pop('name')
     assert result == {
         "credential_type": "Nexus",
```

Comment:
No one is going to look at this and know why "name" was popped off without an assertion, but "credential_type" is in the assertion data.

I would swap the two and add a comment that "credential_type" is only provided in the return data for backward compatibility.

This is important if we declare the RETURNS documentation later.